projects/storefrontapp-e2e-cypress/cypress.config.ts

```diff
@@ -17,7 +17,7 @@ export default defineConfig({
   env: {
     CLIENT_ID: 'mobile_android',
     CLIENT_SECRET: 'secret',
-    API_URL: 'https://40.76.109.9:9002',
+    API_URL: 'https://spartacus-dev-asx1.eastus.cloudapp.azure.com:8443',
```

Comment:
`major:` let's not forget to revert this change

Reply:
reverted